Immutable/Apex.Analyzers.Immutable/Rules/ImmutableTypes.cs

```diff
@@ -0,0 +1,278 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+namespace Apex.Analyzers.Immutable.Rules
+{
+    public class ImmutableTypes
```

Comment:
I'd like to keep this internal until we can move it out into a separate package

Reply:
Fixed: 3b1a856c2032553b869476cc3f7e252bffb706aa